app/views/shared/_individual_progress.html.haml

```diff
@@ -45,9 +45,9 @@
     CONTINUE
 - elsif ridp == true && step == 2
   - if next_link.present?  && current_user.has_hbx_staff_role? && (@person.primary_family.application_type == "Phone" || @person.primary_family.application_type == "Paper")
-    = link_to "Continue", next_link, :method => :get, class: 'button btn btn-lg btn-primary  btn-block interaction-click-control-continue', id: 'btn-continue', :disabled => (application_verified || identity_verified) ? false : true
+    = link_to "Continue", (pundit_allow(@person.consumer_role, :ridp_accessible?) ? next_link : request.path), :method => :get, :class => "button btn btn-lg btn-primary  btn-block interaction-click-control-continue" + pundit_class(@person.consumer_role, :ridp_accessible?), id: 'btn-continue', :disabled => (application_verified || identity_verified) ? false : true
```

Comment:
@ATBull81 The problem here is not just about the Continue button. It is also about the endpoint `help_paying_coverage` not having a pundit policy(including the Identity Verification check). The user can simply change the URL and access this endpoint once they are stuck on the Failed RIDP Verification page. Please add a pundit policy for the endpoint this way any user is blocked if they are not allowed to access the endpoint. Consider all the below while implementing the Pundit Policy.
- Admin Access - `hbx_staff_role.permission.modify_family`
- Broker Access - Brokers are assigned to families to access the Consumer Accounts. We need to check if the Broker's Agency Staff Role is active and is associated with the family.
- Account Access by the Family itself.

cc: @mdkaraman

Reply:
https://github.com/ideacrew/enroll/pull/3573/files#diff-437aeaa6d71572bcc20d9566d96f9918b4a1ef2640a5d6dd60989bc138703d8dR53-R61

Reply:
https://github.com/ideacrew/enroll/pull/3573/files#diff-2a0aa5303b979e3a8f8e489305be1c81a26dfb6538b8510cf1ac4e25e62a2401R311 for blocking endpoint